src/containers/NewProject.js

```diff
@@ -247,26 +324,99 @@ class NewProject extends Component {
       test: testId,
       database: ormId
     }
-    await addProject(session.authToken, newProject)
+
+    const { result: project } = await addProject(session.authToken, newProject)
+    await getProjectFiles(session.authToken, project.id)
+    history.push(`/projects/${project.id}`)
```

Comment:
When I tested this, it didn't redirect. It just stayed at the end of the project creation process. The following error was logged to the console:
```
Unhandled promise rejection 
TypeError: history is undefined
Stack trace:
_callee2$@http://localhost:8080/bundle.js:151368:17
tryCatch@http://localhost:8080/bundle.js:71803:37
invoke@http://localhost:8080/bundle.js:72041:22
defineIteratorMethods/</prototype[method]@http://localhost:8080/bundle.js:71855:16
step@http://localhost:8080/bundle.js:150988:183
step/<@http://localhost:8080/bundle.js:150988:361
run@http://localhost:8080/bundle.js:69810:22
notify/<@http://localhost:8080/bundle.js:69823:30
flush@http://localhost:8080/bundle.js:33092:9
```
Not a very helpful stack trace, but I guess it's...something?